shardingsphere-features/shardingsphere-encrypt/shardingsphere-encrypt-rewrite/src/main/java/org/apache/shardingsphere/encrypt/rewrite/token/generator/impl/EncryptInsertValuesTokenGenerator.java

```diff
@@ -133,7 +133,8 @@ private void encryptToken(final InsertValue insertValueToken, final String table
     private void addPlainColumn(final InsertValue insertValueToken, final int columnIndex,
                                 final String tableName, final String columnName, final InsertValueContext insertValueContext, final Object originalValue) {
         if (getEncryptRule().findPlainColumn(tableName, columnName).isPresent()) {
-            DerivedSimpleExpressionSegment derivedExpressionSegment = insertValueContext.getParameters().isEmpty()
+            DerivedSimpleExpressionSegment derivedExpressionSegment = (insertValueContext.getParameters().isEmpty()
+                    || insertValueContext.getValueExpressions().get(columnIndex) instanceof LiteralExpressionSegment)
```

Comment:
Hi, These codes appear twice, do you think it is better to exact a function for them?

Reply:
Done.